import_flow_events.py

```diff
@@ -54,22 +59,63 @@
       utmTerm VARCHAR(40)
     );
 """
+Q_CREATE_METADATA_TABLE = """
+    CREATE TABLE IF NOT EXISTS flow_metadata (
+      flowId VARCHAR(64) NOT NULL UNIQUE SORTKEY,
+      beginTime TIMESTAMP NOT NULL,
+      duration INTERVAL NOT NULL,
+      completed BOOLEAN NOT NULL,
+      newAccount BOOLEAN NOT NULL,
+      browser VARCHAR(40),
+      browserVersion VARCHAR(40),
+      os VARCHAR(40),
+      context VARCHAR(40),
+      entrypoint VARCHAR(40),
+      migration VARCHAR(40),
+      service VARCHAR(40),
+      utmCampaign VARCHAR(40),
+      utmContents VARCHAR(40),
+      utmMedium VARCHAR(40),
+      utmSource VARCHAR(40),
+      utmTerm VARCHAR(40)
+    );
+"""
+Q_CREATE_EVENTS_TABLE = """
+    CREATE TABLE IF NOT EXISTS flow_events (
+      timestamp TIMESTAMP NOT NULL SORTKEY,
```

Comment:
I've added `timestamp` to this table and made it `SORTKEY`, which is not in the spec. I use `timestamp` as the deletion criteria in `Q_CLEAR_DAY_EVENTS`.

I did briefly delete from that table using an `ON DELETE CASCADE` foreign key reference instead, but then I realised that was error prone in the cases where the `flow.begin` event occurs on a different day to the target event.